concourse/pipelines/guest-package-build.yaml

```diff
@@ -1596,6 +1596,398 @@ jobs:
 
+- name: build-compute-image-windows
+  plan:
+  - get: compute-image-windows
+    trigger: true
+    params:
+      skip_download: true
+  - load_var: commit-sha
+    file: compute-image-windows/.git/ref
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: get-github-token
+    file: guest-test-infra/concourse/tasks/get-github-token.yaml
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: timestamp
+    file: timestamp/timestamp
+  - task: compute-image-windows-integration-test
+    file: guest-test-infra/concourse/tasks/package-gointegtest.yaml
+    vars:
+      repo-name: compute-image-windows
+      timestamp: ((.:timestamp))
+  - task: validate-integtest-result
+    file: guest-test-infra/concourse/tasks/validate-integtest-result.yaml
+    vars:
+      timestamp: ((.:timestamp))
+      repo-name: compute-image-windows
+  - task: compute-image-windows-gotest-coverage
+    file: guest-test-infra/concourse/tasks/gotest.yaml
+    input_mapping:
+      repo: compute-image-windows
+  - load_var: coverage-percent
+    file: gotest/coverage.txt
+  - task: generate-package-version
+    file: guest-test-infra/concourse/tasks/generate-package-version.yaml
+    input_mapping: {repo: compute-image-windows}
+  - load_var: package-version
+    file: package-version/version
+  - task: guest-package-build-compute-image-windows-goo
+    file: guest-test-infra/concourse/tasks/guest-package-build.yaml
+    vars:
+      wf: build_goo.wf.json
+      repo-name: compute-image-windows
+      version: ((.:package-version))
+      gcs_path: gs://gcp-guest-package-uploads/compute-image-windows
+      git_ref: ((.:commit-sha))
+  - put: compute-image-windows-tag
+    params:
+      name: package-version/version
+      tag: package-version/version
+      commitish: compute-image-windows/.git/ref
+  - task: compute-image-windows-publish-coverage
+    file: guest-test-infra/concourse/tasks/publish-coverage.yaml
+    vars:
+      package_name: "google-compute-image-windows"
+      coverage_percent: ((.:coverage-percent))
+  on_success:
+    task: success
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "build-compute-image-windows"
+      result_state: "success"
+      start_timestamp: ((.:start-timestamp-ms))
+  on_failure:
+    task: failure
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "guest-package-build"
+      job: "build-compute-image-windows"
+      result_state: "failure"
+      start_timestamp: ((.:start-timestamp-ms))
+
+- name: upload-windows-certgen-staging
+  plan:
+  - in_parallel:
+      steps:
+      - get: compute-image-windows-tag
+        passed: [build-compute-image-windows]
```

Comment:
usually we set `trigger: true` on the first injection job, wdyt?

Reply:
For the single package repos that makes sense. Since these packages share the repo a build will likely only be for 1 of the packages.